docs/telemetry.md

```diff
@@ -64,15 +64,15 @@ The event ping contains a list of events ([see event format on readthedocs.io](h
 (*) This event is sent at the end of every session.
 
 ### Browser Overlay
-| Event                                  | category | method                | object       | value                    | extras.       |
+| Event                                  | category | method                | object       | value                    | extra.       |
 |----------------------------------------|----------|-----------------------|--------------|--------------------------|---------------|
 | Browser: back clicked                  | action   | click                 | menu         | back                     |               |
 | Browser: forward clicked               | action   | click                 | menu         | forward                  |               |
 | Browser: refresh clicked               | action   | click                 | menu         | refresh                  |               |
 | Turbo mode switch clicked              | action   | change                | turbo_mode   | on/off (the new value)   |               |
 | Pin site switch clicked                | action   | change                | pin_page     | on/off (the new value)   | desktop_mode* |
 | Desktop mode switch clicked            | action   | change                | desktop_mode | on/off (the new value)   |               |
-| Tile clicked                           | action   | click                 | home_tile    | bundled/custom/youtube** | tile_id***    |
+| Tile clicked                           | action   | click                 | home_tile    | bundled/custom/youtube_tile** | tile_id***    |
```

Comment:
This can also be "pocket", right?

Reply:
That's actually part of #2399 - we don't have pocket telemetry clicks in right now.

Reply:
Ah, this had some conflicts which is why this didn't have Pocket - rebased and fixed now.